sklearn/impute/tests/test_impute.py

```diff
@@ -1364,3 +1364,27 @@ def test_imputation_order(order, idx_order):
                                random_state=0).fit(X)
         idx = [x.feat_idx for x in trs.imputation_sequence_]
         assert idx == idx_order
+
+
+@pytest.mark.parametrize(
+    "Imputer,arg",
+    [(SimpleImputer, 'strategy'), (IterativeImputer, 'initial_strategy')])
+@pytest.mark.parametrize(
+    "strategy",
+    ['mean', 'median', 'most_frequent', "constant"])
```

Comment:
Perhaps we should instead include (some of) these different strategies in IMPUTERS in test_common.py??

Reply:
I'm sorry, but i couldn't understand your suggestion